errors.ts

```diff
@@ -99,6 +83,26 @@ export function installUncaughtExceptionListener(actionOnException?: () => void)
 	});
 }
 
+function tryTrackException(error: Error, injector: IInjector): void {
+	let disableAnalytics: boolean;
+	try {
```

Comment:
Doesn't this mean that we have to write our unit tests to mock this value to true and remove this code from here?